5.0/en/0x50-V50-attacks-via-browser.md

```diff
@@ -1,18 +1,16 @@
-# V50 Defense against attacks via browser
+# V50 Browser-based Attacks
 
 note: all category names, section names and file name will be most likely renamed in the future.
 
-The category focuses on requirements which must take down attacks via victim browser and contain requirements which we don't need for verifying machine-to-machine solutions.
-
+The category focuses on requirements which protect against attacks via that are executed via a victim browser. These requirements will not be relevant for machine-to-machine solutions.
 
 ## V50.1 Site Isolation Architecture
 
 | # | Description | L1 | L2 | L3 | CWE | Issue |
 | :---: | :--- | :---: | :---: | :---: | :---: | :---: |
-| **50.1.1** | [ADDED] Verify that separate applications are hosted on different hostnames so as to benefit from the protections provided by the "same origin policy" and the hostname restrictions on cookies. | ✓ | ✓ | ✓ | 668 | [#1299](https://github.com/OWASP/ASVS/issues/1299) |
-
+| **50.1.1** | REMOVE SECTION. | | | | | |
```

Comment:
I have moved this requirement to the origin separation section. because I think all requirements related to origin should be together

Reply:
I don't agree with the move. This requirement also for Same-Site policy (for cookies) and it's not Origin specific.

Maybe useful comment from the related issue: https://github.com/OWASP/ASVS/issues/1299#issuecomment-1167183581

Reply:
Same Site is also an origin thing because (roughly) it defines that cookies will only be sent on a same origin request. To me, the comment you posted strengthens my perspective that this is an origin thing.

Reply:
No no no, and another no. Site and Origin are different things, and SameSite does NOT mean that cookies are sent to only to Same-Origin, but also to Cross-Origin IF those are matching with Same-Site.

Reply:
Change reverted